### DISCUSSION TITLE
New paper, Lopez-Corredoira, Melia: Age of Massive Galaxies at redshift 8

### DISCUSSION THREAD

**Discussion by mikehelland** [a-cosmology-group/acg#220] [Astrophysics and Physics]
https://arxiv.org/abs/2405.12665

**Age of massive galaxies at redshift 8**
M. Lopez-Corredoira, F. Melia, J.-J. Wei, C.-Y. Gao

**Comment by HanDeBruijn** [comment]
From the [paper](https://arxiv.org/pdf/2405.12665) we quote:
>Other cosmological scenarios might account for these observations better than &Lambda;CDM, however. For example, the $R_h = ct$ universe ([Melia &amp; Shevchuk](https://www.researchgate.net/publication/229056097_The_Rh_ct_universe) 2012; Melia 2020; 2023) gives an age of the Universe of 14.6 Gyr, or Covarying Coupling Constants + Tired-light (CCC+TL) hybrid model ([Gupta](https://github.com/orgs/a-cosmology-group/discussions/102) 2023) gives an age of the Universe of 26.7 Gyr (though difficult to reconcile with the age of the oldest known sources, in our opinion). In the most reasonable alternative, i.e., the $R_h = ct$ universe, the timeline at $z \gtrapprox 6$ is approximately twice as long as that in the standard model, assuming $H_0 = 70\mbox{ }km\mbox{ }s^{−1}Mpc^{−1}$.

Isn't that a bit predictable? [Fulvio Melia](https://en.wikipedia.org/wiki/Fulvio_Melia) as a co-author of this paper, and his favorite $R_h = ct$ universe model turns out to be _the most reasonable alternative_? Needless to say that I know of at least one alternative universe model with a timeline _approximately twice as long as that in the standard model_, namely the [Variable Mass Theory](https://www.youtube.com/watch?v=T7jVviVBzKs), where the age of the universe (in orbital time) is $2/H_0 \approx 28.8$ Gyr.

**Comment by RedshiftDrift** [comment]
Thanks @mikehelland, this is a good paper.  Unfortunately Lopez-Corredoira et al. always have to downplay their arguments with statements like "our conclusions result from several approximations in stellar astrophysics and extinction, so they should be taken with a grain of salt", but I guess that's needed if they want to publish past the referees.

**Comment by AngelBlanco19** [comment]
@mikehelland - Infinity and eternity has no age by definition. Matter is inexhaustible, infinite and eternal. Here is the irrefutable proof, which was hidden about 2316 year or so: «The Physical Mathematics and Geometry of Dialectical Materialism Versus the Euclidean "Mathematics" and "Geometry" of Philosophical Idealism» [https://vixra.org/abs/2402.0116](https://vixra.org/abs/2402.0116)

Every finite, determinate and countable magnitude or physical object is a unique and exclusive product of the physical process of Time, the most important of Matter.
Every gravitational field and celestial body with gravity (except its nucleus) is a flat, finite, determinate and countable sphere, with a certain max. physical limit.
The redshift depends on the size of the gravitational field where the light source is located, and therefore on its age, as well as on the distance to which it is and the age of all the gravitational fields that it has to cross to the observer. Therefore it depends on the direction in which it is measured. Here's something on the subject to get you started: «The Gravitational Paradox and the Proposed Experiment to Demonstrate Gravitational Distortion of "Spacetime"» [https://vixra.org/abs/2202.0042](https://vixra.org/abs/2202.0042)
